internal/cleanup/service_logs.go

```diff
@@ -26,5 +31,33 @@ func ServiceLogs() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("can't remove content (path: %s): %w", locationManager.ServiceLogDir(), err)
 	}
+
 	return locationManager.ServiceLogDir(), nil
 }
+
+// ServiceLogs function removes service logs from temporary directory in the `~/.elastic-package`.
+func ServiceLogsIndependentAgents(profile *profile.Profile) (string, error) {
+	logger.Debug("Clean all service logs from independent Elastic Agents")
+
+	packageRootPath, err := packages.MustFindPackageRoot()
+	if err != nil {
+		return "", fmt.Errorf("locating package root failed: %w", err)
+	}
+
+	packageFolder := filepath.Base(packageRootPath)
+	serviceLogsDir := agentdeployer.ServiceLogsDir(profile)
+
+	folderGlob := fmt.Sprintf("%s/agent-%s-*", serviceLogsDir, packageFolder)
```

Comment:
As this package removes the built packages of the given package (folder), I tried to remove just the service logs folders for the given package.

Reply:
Do we use the name of the directory for these logs directories or the name of the package in the manifest? (There are packages with names different to their directories).
Maybe we could have a function to obtain this directory path, so we use the same when creating and when deleting it.

Reply:
Looking at the code, it was the folder package name what it was used.
But it is true, that this was not clear and easy to know from the code.

I tried to refactor the code a little in https://github.com/elastic/elastic-package/pull/2033/commits/031c2cc2cacab818e46ff2d934727ed4df721aab to help about that.